### PR TITLE
testing for register and profile

### PR DESCRIPTION
**Pull request by mnsison**

### CODE REVIEW

File: src/Profile/profile.spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import EditProfile from './EditProfile'
+import "@testing-library/jest-dom/extend-expect";
+import {BrowserRouter as Router} from 'react-router-dom';
```
Comment (echanlee) [inline src/Profile/profile.spec.js]: do you use this import?

**Review by echanlee — COMMENTED** [review]
Is it possible to add the testing files into a separate folder (maybe in test folder add a folder that specifies jest tests if it's possible)

File: src/Profile/profile.spec.js (hunk 1 above)
Comment (mnsison) [reply src/Profile/profile.spec.js]: oh i forgot the router part is only for the register page let me remove that

Comment (mnsison) [reply src/Profile/profile.spec.js]: also I tried moving the test files to the Tests folder but the tests don't run and I made sure that I was importing properly

**Review by echanlee — APPROVED** [review]

**Pull request merged**